src/getHumanAge.test.js

```diff
@@ -7,4 +7,44 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test('should be return 0 if catAge or dogAge is equal 0', () => {
+    expect(getHumanAge(0, 0))
+      .toEqual([0, 0]);
+  });
+
+  test('should be return a right value for first 14 cat & dog age', () => {
```

Comment:
The first test (above) is fixed and looks really good. Now to all the rest tests:
please try to imagine, that this test fails. It will be difficult to understand which is the "right value".
Feel free to write "should return 0 if ...", "should return [1, 1] if..."

Reply:
fixed